pkg/api/controllers/projectconfig/prebuild/process_git_event.go

```diff
@@ -17,7 +17,7 @@ import (
 //	@Tags			prebuild
 //	@Summary		ProcessGitEvent
 //	@Description	ProcessGitEvent
-//	@Param			workspace	body	interface{}	true	"Webhook event"
+//	@Param			body	body	interface{}	true	"Webhook event"
```

Comment:
Why was it 'workspace' here but now it is 'body'

Reply:
"workspace" was a copy/paste error before